common/djangoapps/util/testing.py

```diff
@@ -1,9 +1,18 @@
 import sys
 
+from datetime import datetime
+from pytz import UTC
 from mock import patch
 
 from django.conf import settings
 from django.core.urlresolvers import clear_url_caches, resolve
+from openedx.core.djangoapps.course_groups.models import CourseUserGroupPartitionGroup
```

Comment:
I think I'm unsure of what's the correct way of handling things when you have dependencies on both `openedx` and `common`. How should we be doing things in that case?